docs/api.asciidoc

```diff
@@ -75,18 +80,22 @@ Arguments:
 
   * `name`: A name for your transaction. Transactions are grouped by name. **Required**.
   * `type`: A `type` for your transaction eg. `db.postgresql.sql`.
-  * `rack_env: env`: An optional Rack `env` used to enrich the transaction with information about the current request.
-  * `&block`: An optional block to wrap with the transaction. The block is passed the transaction as an optional argument.
+  * `context:`: An optional Context used to enrich the transaction with
+  information about the current request.
+  * `&block`: An optional block to wrap with the transaction. The block is passed the
+  transaction as an optional argument.
 
 Returns the transaction.
 
 [float]
 [[api-span]]
 ==== `ElasticAPM.span`
 
-Most transactions have nested spans signifying work of a specific sort eg. database queries or external web requests.
+Most transactions have nested spans signifying work of a specific sort eg. database
+queries or external web requests.
 
-If given a block, wrap the code inside in a span. If not you are expected to close the span yourself with `span.done(result)`.
+If given a block, wrap the code inside in a span.
```

Comment:
Could you align the formulation with the transaction:

> If given a block, the code inside will be wrapped in a transaction.

Could you generally add a hint which spans are automatically collected, and also relate this to manually trace transactions.